account-kit/java/signer-sdk/pom.xml

```diff
@@ -7,16 +7,27 @@
   <description>This is an Alchemy signer SDK</description>
   <url>https://github.com/alchemyplatform/aa-sdk/tree/main/account-kit/java/signer-sdk</url>
 
-  <groupId>io.github.andysim3d</groupId>
+  <groupId>com.alchemy.accountkit</groupId>
   <artifactId>signer-sdk</artifactId>
-  <version>1.1-SNAPSHOT</version>
+  <version>0.1.0-alpha-75d8e542</version>
```

Comment:
why put a hash here?

Reply:
form Dan - he want to use a git sha to quick find the build.